tests/topotests/conftest.py

```diff
@@ -522,9 +522,14 @@ def pytest_configure(config):
         is_xdist = True
         is_worker = True
 
-    resource.setrlimit(
-        resource.RLIMIT_CORE, (resource.RLIM_INFINITY, resource.RLIM_INFINITY)
-    )
+    try:
+        resource.setrlimit(
+            resource.RLIMIT_CORE, (resource.RLIM_INFINITY, resource.RLIM_INFINITY)
+        )
+    except ValueError:
+        # The hard limit cannot be raised. Raise the soft limit to previous hard limit
+        core_rlimits = resource.getrlimit(resource.RLIMIT_CORE)
+        resource.setrlimit(resource.RLIMIT_CORE, (core_rlimits[1], core_rlimits[1]))
```

Comment:
Btw, isn't just enough to `pass` here without re-setting what is already set (by the system)?